src/main/java/com/google/api/codegen/util/php/PhpNameFormatter.java

```diff
@@ -17,15 +17,24 @@
 import com.google.api.codegen.util.Name;
 import com.google.api.codegen.util.NameFormatter;
 import com.google.api.codegen.util.NamePath;
+import com.google.common.collect.ImmutableSet;
 
 /**
  * The NameFormatter for PHP.
  */
 public class PhpNameFormatter implements NameFormatter {
 
+  private String wrapIfKeywordOrBuiltIn(String name) {
+    if (KEYWORD_BUILT_IN_SET.contains(name)) {
```

Comment:
So PHP is weird in that it treats any casing of a keyword as using that keyword. Examples: `empty`, `Empty`, `EMPTY`. So, you need to do a `toLower()` on the input `name`. Also, there are more functions in this class that need `wrapIfKeywordOrBuiltIn` applied.